pcp/Platform.h

```diff
@@ -0,0 +1,257 @@
+#ifndef HEADER_Platform
+#define HEADER_Platform
+/*
+htop - pcp/Platform.h
+(C) 2014 Hisham H. Muhammad
+(C) 2020-2021 htop dev team
+(C) 2020-2021 Red Hat, Inc.  All Rights Reserved.
+Released under the GNU GPLv2, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include <ctype.h>
+#include <stdbool.h>
+#include <pcp/pmapi.h>
+
+#undef PACKAGE_URL
+#undef PACKAGE_NAME
+#undef PACKAGE_STRING
+#undef PACKAGE_TARNAME
+#undef PACKAGE_VERSION
+#undef PACKAGE_BUGREPORT
```

Comment:
Why `#undef` all those defines? Should be commented/documented. Especially as this is in the middle of all the includes.

Reply:
We have a conflict with macros of the same name from PCP config.h - I'll add a comment to that effect.